gateway.yml

```diff
@@ -1,10 +1,43 @@
-baseUri: http://0.0.0.0:8090/radar-gateway/
-managementPortalUrl: http://localhost:8080/managementportal
-schemaRegistryUrl: http://localhost:8081
-restProxyUrl: http://localhost:8082
-resourceConfig: org.radarbase.gateway.inject.EcdsaJwtGatewayResources
-jwtKeystorePath: src/integrationTest/docker/etc/config/keystore.p12
-jwtKeystoreAlias: radarbase-managementportal-ec
-jwtKeystorePassword: radarbase
-jwtECPublicKeys: []
-jwtRSAPublicKeys: []
+# Resource config class
+#resourceConfig: org.radarbase.gateway.inject.ManagementPortalEnhancerFactory
+
+server:
+    # URI to serve data to
+    baseUri: http://0.0.0.0:8090/radar-gateway/
+    # Maximum number of simultaneous requests to Kafka.
+    #maxRequests: 200
+    # Maximum request content length, also when decompressed.
+    # This protects against memory overflows.
+    #maxRequestSize: 25165824
+    # Whether JMX should be enabled. Disable if not needed, for higher performance.
+    #isJmxEnabled: true
+
+kafka:
+    # Number of Kafka brokers to keep in a pool for reuse in multiple requests.
+    # poolSize: 20
+    # Kafka producer settings. Read from https://kafka.apache.org/documentation/#producerconfigs.
+    producer:
+        bootstrap.servers: kafka-1:9092
+    # Kafka Admin Client settings. Read from https://kafka.apache.org/documentation/#adminclientconfigs.
+    # admin: {}
```

Comment:
is this a map? how do we specify multiple properties?

Reply:
Indeed, the same as for `producer`